tests/Instances.hs

```diff
@@ -145,3 +149,12 @@ instance Arbitrary Version where
 makeVersion :: [Int] -> Version
 makeVersion b = Version b []
 #endif
+
+instance Arbitrary (Proxy a) where
+    arbitrary = pure Proxy
+
+instance Arbitrary b => Arbitrary (Tagged a b) where
+    arbitrary = Tagged <$> arbitrary
+
+instance Arbitrary a => Arbitrary (Const a b) where
+    arbitrary = Const <$> arbitrary
```

Comment:
Also some of my other PR's restricts `QuickCheck` to `2.8.3` because of others orphan instances